docs/contributor/image_mode.md

```diff
@@ -0,0 +1,308 @@
+
+### Run Container
+
+Run the following command to start the MicroShift `bootc` image in an interactive
+terminal session. Note that the host container storage and kernel modules paths
+are shared with the container.
+
+```
+IMAGE_NAME=microshift-4.16-bootc
+
+sudo podman run --rm -it --privileged \
+    -v /var/lib/containers/storage:/var/lib/containers/storage \
+    -v /lib/modules:/lib/modules:ro \
+    --name "${IMAGE_NAME}" \
+    "${IMAGE_NAME}"
+```
+
+> If additional LVM volume group device was allocated as described in the
+> [Configure CSI](#configure-csi) section, the loop device should automatically
+> be shared with the container and used by the MicroShift CSI driver.
+
+Log into the container host when prompted using the `redhat:redhat` credentials.
+Run the following command to verify that all the MicroShift pods are up and running
+without errors.
+
+```
+sudo oc get pods -A -w \
+    --kubeconfig /var/lib/microshift/resources/kubeadmin/kubeconfig
+```
+
+> Run the `sudo shutdown now` command to stop the container host.
+
+## Run MicroShift Bootc Virtual Machine
+
+Log into the `physical hypervisor host` using the user credentials that have
+SUDO permissions configured.
+
+### Prepare Kickstart File
+
+Create the `kickstart.ks` file with the following contents to be used during the
+virtual machine installation.
+
+```
+lang en_US.UTF-8
+keyboard us
+timezone UTC
+text
+reboot
+
+# Partition disk with a 1GB boot XFS partition and an LVM volume containing a 10GB+
+# system root. The remainder of the volume is for the CSI driver for storing data.
+zerombr
+clearpart --all --initlabel
+part biosboot --fstype=biosboot --size=1 --asprimary
+part /boot/efi --fstype=efi --size=200
+part /boot --fstype=xfs --asprimary --size=800
+part pv.01 --grow
+volgroup rhel pv.01
+logvol / --vgname=rhel --fstype=xfs --size=10240 --name=root
+
+# Lock root user account
+rootpw --lock
+
+# Configure network to use DHCP and activate on boot
+network --bootproto=dhcp --device=link --activate --onboot=on
+
+# Pull a `bootc` images from a remote registry
+ostreecontainer --url quay.io/myorg/mypath/microshift-4.16-bootc --no-signature-verification
+```
+
+The kickstart file uses a special [ostreecontainer](https://pykickstart.readthedocs.io/en/latest/kickstart-docs.html#ostreecontainer)
+directive to pull a `bootc` image from the remote registry and install it on top
+of the base RHEL operating system.
+
+> Replace `myorg/mypath` with your remote registry organization name and path.
+
+### Create Virtual Machine
+
+Download a RHEL boot ISO image from https://developers.redhat.com/products/rhel/download.
+Copy the downloaded file to the `/var/lib/libvirt/images` directory.
+
+Run the following commands to create a RHEL virtual machine with 2 cores, 2GB of
+RAM and 20GB of storage. The command uses the kickstart file prepared in the
+previous step to pull a `bootc` image from the remote registry and install it on
+top of the base RHEL operating system.
```

Comment:
Same comment as above re: "on top of the base RHEL operating system"

Reply:
Reworded. Is this better?